ChangeLog

```diff
@@ -21,7 +21,7 @@ needrestart 3.6
     - [Core] Fix verbose/verbosity confusion in needrestart.conf.
       (github pull request #197 by Jan-Philipp Litza @jplitza)
 
- -- Thomas Liske <thomas@fiasko-nw.net>  
```

Comment:
This is intentional, the date is appended during release.

Reply:
Apologies, I thought I removed that part of the commit already, it slipped through.